packages/lesswrong/components/languageModels/LanguageModelLauncherButton.tsx

```diff
@@ -57,7 +57,7 @@ export const LanguageModelLauncherButton = ({classes}: {
   },[openDialog, captureEvent, setCookie]);
 
   useEffect(() => {
-    if (cookies[SHOW_LLM_CHAT_COOKIE]!=="true") {
+    if (cookies[SHOW_LLM_CHAT_COOKIE]==="false") {
```

Comment:
This seems like an inversion you don't want?

Reply:
No, it is. For users I'm currently adding, I want it to pop open if there is no cookie saying otherwise.

Reply:
I don't think the button in the bottom right corner is that noticeable